extension/WebJobs.Extensions.RabbitMQ/Config/RabbitMQExtensionConfigProvider.cs

```diff
@@ -37,12 +37,11 @@ public RabbitMQExtensionConfigProvider(IOptions<RabbitMQOptions> options, INameR
 
         public void Initialize(ExtensionConfigContext context)
         {
-            if (context == null)
-            {
-                throw new ArgumentNullException("context");
-            }
+            _ = context ?? throw new ArgumentNullException(nameof(context));
 
-            var rule = context.AddBindingRule<RabbitMQAttribute>();
+#pragma warning disable 0618
+            FluentBindingRule<RabbitMQAttribute> rule = context.AddBindingRule<RabbitMQAttribute>();
+#pragma warning restore 0618
```

Comment:
`FluentBindingRule` seems to be obsolete. I will check sometime in future why this is obsoleted and if there's a non-obsoleted replacement for it, I will switch the variable type accordingly.